ev3dev.py

```diff
@@ -27,12 +27,16 @@
 
 #~autogen
 
+import os
 import os.path
 import fnmatch
 import numbers
 import platform
 import fcntl
 import array
+import mmap
+import ctypes
+from PIL import Image, ImageDraw
```

Comment:
This raises an error on my system - I don't have PIL by default - do we need to add this to the base image? Or does the latest image already have PIL?